sphinx/writers/latex.py

```diff
@@ -45,6 +45,73 @@
 URI_SCHEMES = ('mailto:', 'http:', 'https:', 'ftp:')
 SECNUMDEPTH = 3
 
+DEFAULT_SETTINGS = {
+    'papersize':       'letterpaper',
+    'pointsize':       '10pt',
+    'pxunit':          '49336sp',
+    'classoptions':    '',
+    'extraclassoptions': '',
+    'passoptionstopackages': '',
+    'geometry':        '\\usepackage[margin=1in,marginparwidth=0.5in]'
+                       '{geometry}',
+    'inputenc':        '',
+    'utf8extra':       ('\\ifdefined\\DeclareUnicodeCharacter\n'
+                        '  \\DeclareUnicodeCharacter{00A0}{\\nobreakspace}\n'
+                        '\\fi'),
+    'cmappkg':         '\\usepackage{cmap}',
+    'fontenc':         '\\usepackage[T1]{fontenc}',
+    'amsmath':         '\\usepackage{amsmath,amssymb,amstext}',
+    'babel':           '\\usepackage{babel}',
+    'fontpkg':         '\\usepackage{times}',
+    'fncychap':        '\\usepackage[Bjarne]{fncychap}',
+    'longtable':       '\\usepackage{longtable}',
+    'hyperref':        ('% Include hyperref last.\n'
+                        '\\usepackage[colorlinks,breaklinks,%\n'
+                        '            '
+                        'linkcolor=InnerLinkColor,filecolor=OuterLinkColor,%\n'
+                        '            '
+                        'menucolor=OuterLinkColor,urlcolor=OuterLinkColor,%\n'
+                        '            '
+                        'citecolor=InnerLinkColor]{hyperref}\n'
+                        '% Fix anchor placement for figures with captions.\n'
+                        '\\usepackage{hypcap}% it must be loaded after hyperref.\n'
+                        '% Set up styles of URL: it should be placed after hyperref.\n'
+                        '\\urlstyle{same}'),
+    'usepackages':     '',
+    'numfig_format':   '',
+    'contentsname':    '',
+    'preamble':        '',
+    'title':           '',
+    'date':            '',
+    'release':         '',
+    'author':          '',
+    'logo':            '',
+    'releasename':     'Release',
+    'makeindex':       '\\makeindex',
+    'shorthandoff':    '',
+    'maketitle':       '\\maketitle',
+    'tableofcontents': '\\sphinxtableofcontents',
+    'postamble':       '',
+    'printindex':      '\\printindex',
+    'transition':      '\n\n\\bigskip\\hrule{}\\bigskip\n\n',
+    'figure_align':    'htbp',
+    'tocdepth':        '',
+    'secnumdepth':     '',
+    'pageautorefname': '',
+}
+
+ADDITIONAL_SETTINGS = {
+    'pdflatex': {
+        'inputenc':     '\\usepackage[utf8]{inputenc}',
+    },
+    'xelatex': {
+        'babel':        '',  # disable babel
+        'inputenc':     '\\usepackage{polyglossia}',
```

Comment:
As babel and polyglossia are alternatives, and as polyglossia is not related to the input encoding I feel it makes more sense to put the `'\\usepackage{polyglossia}'`as value of the `'babel'`key and leave the`'inputenc'` key with empty string value.